userguide/cli.rst

```diff
@@ -985,6 +985,17 @@ To collect the output of every module, use :literal:`-A`:
 
    freenas-debug -A
 
+For collecting debug information about a single pool, use
+:command:`zdb` with :literal:`-U /data/zfs/zpool.cache`
+followed by the name of the pool:
+
+.. code-block:: none
+
+    zdb -U /data/zfs/zpool.cache pool1
```

Comment:
In master, I think the example volume name is "volume1". Please check.

Reply:
Actually pool1 is used much more often than volume1 apparently

Reply:
Changed it to volume1 anyway.